README.md

```diff
@@ -0,0 +1,63 @@
+# Hello-Microverse
+
+> In this project, I will copy a project from "Microverse collaboration Trials". No complex coding is required for this exercise. 
+My goal here update the code to meet microverse standards.
+
+
+
+
+
+## Built With
+
+- Html
+- CSS
+- Bootstrap
+- No Technologies used
+
+## Live Demo
+No Demo
+
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+Clone this Repository
```

Comment:
- [ ] It would be better if we specify how we can clone the project, for example, `git clone https://github.com/adamteddychang/Microverse_trials_review.git`. This would help non-technical people also interact with the code or application

Reply:
I have added a more detailed description as to how to clone the project, thank you.